Goobi/plugins/opac/PicaPlugin/org/goobi/production/plugin/CataloguePlugin/PicaPlugin/ConfigOpac.java

```diff
@@ -191,13 +191,6 @@ private static ConfigOpacDoctype getDoctypeByName(String inTitle) {
 			String title = getConfig().getString("doctypes.type(" + i + ")[@title]");
 			if (title.equals(inTitle)) {
 				/* Sprachen erfassen */
-				HashMap<String, String> labels = new HashMap<String, String>();
```

Comment:
Please remove the comment "Sprache erfassen" as this comment is not longer needed.